HISTORY

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+1.1.2 (2016-06-29)
+------------------
+
+* Pinned to djangocms-attributes-field v0.1.0
+* Fixed model name used in migration
```

Comment:
@mkoistinen could you add "* Fixed an issue with folder plugin in Django 1.9" (see #245)